addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -181,6 +224,8 @@ export class SearchBar extends Component {
     async computeSubItems(searchItemId, query) {
         const searchItem = this.searchItems.find((i) => i.id === searchItemId);
         const field = this.fields[searchItem.fieldName];
+        const operator = searchItem.operator || "=";
```

Comment:
I think you left some unnecessary changes in this method, should be cleaned to avoid thinking we modified it.